src/boot/efi/vmm.c

```diff
@@ -187,6 +187,26 @@ typedef struct {
         uint8_t bios_characteristics_ext[2];
 } _packed_ SmbiosTableType0;
 
+typedef struct {
+        SmbiosHeader header;
+        uint8_t manufacturer;
+        uint8_t product_name;
+        uint8_t version;
+        uint8_t serial_number;
+        EFI_GUID uuid;
+        uint8_t wake_up_type;
+        uint8_t sku_number;
+        uint8_t family;
+} _packed_ SmbiosTableType1;
+
+typedef struct {
+        SmbiosHeader header;
+        uint8_t manufacturer;
+        uint8_t product_name;
+        uint8_t version;
+        uint8_t serial_number;
+} _packed_ SmbiosTableType2;
```

Comment:
please move all smbios stuff into a a new pair smbios.[ch]. after all what you are doing here has nothing to do with vmms

Reply:
Should I also move table 11?

Reply:
Moved everything. Maybe worth it to create a separate PR